Dockerfile

```diff
@@ -62,10 +63,12 @@ RUN set -ex \
     && pip install pyasn1 \
     && pip install google-api-python-client \
     && pip install pandas-gbq \
-    && pip install https://github.com/apache/incubator-airflow/archive/1.9.0.tar.gz \
+    && pip install -U setuptools \
+    && pip install https://github.com/apache/incubator-airflow/archive/1.10.1.tar.gz \
+    && pip install snakebite \
     && pip install psycopg2 \
     && pip install celery[redis]==3.1.17 \
-    && pip install https://codeload.github.com/GlobalFishingWatch/pipe-tools/tar.gz/v0.1.6 \
+    && pip install https://codeload.github.com/GlobalFishingWatch/pipe-tools/tar.gz/v1.0.0 \
```

Comment:
@smpiano Add a variable `PIPE_TOOLS_VERSION` and set it to `1.0.0`:
`&& pip install https://codeload.github.com/GlobalFishingWatch/pipe-tools/tar.gz/${PIPE_TOOLS_VERSION}`